packages/components/stamp/src/stamp.tsx

```diff
@@ -164,6 +165,13 @@ const Stamp = (props: Props) => {
       color: getIconColor(props, overrideTextColor),
     });
 
+  useWarnDeprecatedProp(
```

Comment:
I'm taking back the warning about the `children` prop in the `Stamp` component which now will be easier to silence in the MC frontend.